### ISSUE TITLE
Missing type or namespace errors on build

### ISSUE THREAD

**Issue by jackhart591** [bansheeGz/BGCurve#21]
Hello,

When attempting to build my project, I get about 150 missing type or namespace errors. Here is a screenshot:
![Capture](https://user-images.githubusercontent.com/36282715/129497059-3f420303-92e4-4997-9311-3d811e45f890.JPG)

I'm just using what was uploaded to Unity Asset store and I'm on Unity version 2020.3.13f1. Do you have any idea what may be causing this? Thanks.

**Comment by banshee-gzzz** [comment]
Hello

Are all these messages related to BGCurve, or there are some messages, related to other assets?

**Comment by jackhart591** [comment]
Yes, they are all related to BGCurve

**Comment by banshee-gzzz** [comment]
Please, do the following:
1) Make a copy of your project. 
All actions beneath are related to this copy, not the actual project.
Do not change anything in your project, perform all actions on the copy

2) Locate the "Library" folder under the newly copied project root and delete it.
Open the copied project in Unity and try to build it

3) If the errors are still there - please, send me your copied project
Delete all game content you do not want to share before sending

Thank you

**Comment by jackhart591** [comment]
Hello, 

Sorry about the late response. I tried the fix you suggested, it does not work so I asked about sharing it and unfortunately we are not able to share any of the project no matter what we cut out. Because of this, we have decided to remove the spline library from our project because it is not necessary for us with how we currently handle what we were using it for previously. 

Thank you for the timely assistance even if we didn't get it figured out.
